python/ctsm/test/test_sys_fsurdat_modifier.py

```diff
@@ -59,6 +59,32 @@ def test_minimalInfo(self):
         self.assertTrue(fsurdat_out_data.equals(fsurdat_in_data))
 
 
+    def test_crop(self):
+        """
+        This version replances the vegetation with a crop
```

Comment:
spelling

Reply:
Also add comments to explain what's happening, starting with setup and making a cfg file etc...